content/tsconfig.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
As far as I can tell, the issue that this file aims to work around still isn't fixed, so it may be prudent to keep it around for now.

Reply:
I put it back